verbcalc/tests/test_translator.py

```diff
@@ -29,6 +29,16 @@ def test_translation(self):
                   verbcalc.translate('2 mod 2')]
         self.assertListEqual(self.expected, values)
 
+    def test_translation2(self):
```

Comment:
Let's make that name more descriptive
```suggestion
    def test_translation_with_literal_numbers(self):
```